Add HTAN ISB-CGC Release 5 Notebooks

clarisse-lau opened the pull request:
Adding 3 new notebooks to go along with the recent release of HTAN BigQuery tables in ISB-CGC

PozhidayevaDarya committed:
Cell Typist Notebook

PozhidayevaDarya committed:
Delete HTAN/Python Notebooks/Analyzing_HTAN_scRNASeq_Data_Using_CellTypist.ipynb

PozhidayevaDarya committed:
cell_typist_notebook

PozhidayevaDarya committed:
Add files via upload

PozhidayevaDarya committed:
Update Analyzing_HTAN_scRNASeq_Data_Using_CellTypist.ipynb

clarisse-lau committed:
add data guide and drs manifest notebooks

clarisse-lau committed:
Merge pull request #9 from ncihtan/celltypist_notebook

Add CellTypist Notebook

clarisse-lau committed:
Merge pull request #10 from ncihtan/clarisse-r5-notebooks

add data guide and drs manifest notebooks

boaguilar commented:
Hi Clarisse here are some suggestions. 

The link to "alternative authentication methods" in the notebooks  do not really contain authentication information. Perhaps this page is better ( https://cloud.google.com/docs/authentication ) ?

Notebooks: A_Guide_to_HTAN_Data.ipynb
- The title of the notebook ( An Overview of Assay Data in HTAN  ) is a little too different than the name of the file (  A_Guide_to_HTAN_Data.ipynb ) .
- Section 5.2:  It should be Release 5.0 now, the notebook is not really using any r4 table.
- Section 1.3:  r4 tables are not used at all in the notebook. Should we replace r4 by r5  in the text? All the tables in this notebook are now r5.

Notebook: Analyzing_HTAN_scRNASeq_Data_Using_CellTypist.ipynb
- Should we use the versioned tables instead of *_current ones ? 

Creating_CDS_Data_Import_Manifests_Using_BQ.ipynb
- The [NCI Cancer Data Service](https://colab.research.google.com/drive/1hx_LYNX8h0IRIrPOZkSKUt4sDtYWeB3B#scrollTo=cu1lQ8rb20gT) link (at the very end) takes it to a colab notebook that is not publicly accessible.
- Section 1.3  r4 is not used at all in this notebook. Should we replace r4 by r5  in the text? All the tables in this notebook are now r5.

clarisse-lau committed:
update title and versioned table reference

clarisse-lau committed:
update link and versioned table reference

clarisse-lau committed:
update google auth link for new notebooks and template

clarisse-lau committed:
title change and edits

clarisse-lau commented:
Thank you, Boris! I've updated the link to the authentication section in all new notebooks. This link was pulled in from our template, so was in all notebooks. 

It is fine that the CellTypist notebook uses current tables. It uses one MSK single cell BigQuery table that is not expected to change. 

Also changed the r4 references to r5 in section 1.3. 
For section 5.2, Release 4.0 is intended (per the note, our CDS release is currently only up to HTAN data release 4.0).

clarisse-lau committed:
fix formatting
